src/main/java/hudson/remoting/PreloadJarTask.java

```diff
@@ -23,6 +23,7 @@
  */
 package hudson.remoting;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
```

Comment:
There are other unused imports in the ticket. Will just push it later during the bulk update